### ISSUE TITLE
merge GO:0042476 ! odontogenesis and GO:0042475 ! odontogenesis of dentin-containing tooth

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11834]
odontogenesis has two subclasses:

```
     is_a GO:0042476 ! odontogenesis
      is_a GO:0042475 ! odontogenesis of dentin-containing tooth *** 
      is_a GO:0061648 ! tooth replacement
```

The two subclasses are only-childs within orthogonal classification schemes, bad smell.

But focusing on the first. "odontogenesis" is intended to cover such diverse structures as the radula of molluscs and vertebrate teeth. It's not clear if these processes have much in common.

The resulting lattice is misleading and incomplete. For example, "tooth mineralization" is a part of "odontogenesis", but there is no "tooth mineralization of dentin-containing tooth". If we are serious about the species-specific design pattern, then we should implement it consistently.

However, I think this is pointless complexity. Currently all annotations to `odontogenesis` and descendants are vertebrate, and hence necessarily refer to "dentin-containing tooth", and not a chiton "tooth" nor a sea cucumber anal "tooth". However, the taxon-lattice has created confusion, and some groups like ZFIN annotate to "odontogenesis" and others to the more specific "odontogenesis of dentin-containing tooth", leading to the familiar drunken christmas tree annotation arrangement we see with many GO annotations.

One solution would be to make "odontogenesis" a do-not-annotate term and force people to use a specific subclass, including "odontogenesis of radular tooth" and so on when necessary.

But given we only have vertebrate odontogenesis for now, I suggest we simply merge up.

If and when the time comes when we start annotating biomineralization in other species like chitons (I would *love* this), we would create a more specific less ambiguous class and *possibly* introduce a new grouping class.

Note also the grouping as it currently stands doesn't work for interesting commonalities between genuinely homologous structures, such as the [odontode scales](http://purl.obolibrary.org/obo/UBERON_0000041) of Chondrichthyes. 

Reported by: cmungall

Original Ticket: [geneontology/ontology-requests/11670](https://sourceforge.net/p/geneontology/ontology-requests/11670)

**Comment by ukemi** [comment]
I just went to do this merge and noticed that these terms now have logical definitions with different UBERON definitions. So now I am thinking the merge is not such a good idea. I also couldn't seem to select the odontogenesis term in obo-edit. Every time I tried, it froze. Now what I suggest is that once I have figured out what is going on with obo-edit, we just modify the definitions of these terms to correspond to UBERON. Chris?
